client/components/forms/form-country-select/index.jsx

```diff
@@ -1,30 +1,35 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	isEmpty = require( 'lodash/isEmpty' ),
-	classnames = require( 'classnames' ),
-	observe = require( 'lib/mixins/data-observe' ),
-	omit = require( 'lodash/omit' );
+import React from 'react';
+import isEmpty from 'lodash/isEmpty';
+import classnames from 'classnames';
+import observe from 'lib/mixins/data-observe';
+import omit from 'lodash/omit';
+import { localize } from 'i18n-calypso';
 
-module.exports = React.createClass( {
+export default localize( React.createClass( {
 
 	displayName: 'FormCountrySelect',
 
 	mixins: [ observe( 'countriesList' ) ],
 
-	render: function() {
-		var countriesList = this.props.countriesList.get(),
-			options = [];
-
+	getOptions( countriesList ) {
 		if ( isEmpty( countriesList ) ) {
-			options.push( { key: '', label: this.translate( 'Loading…' ), disabled: 'disabled' } );
-		} else {
-			options = options.concat( countriesList.map( function( country ) {
-					return { key: country.code, label: country.name };
-				}
-			) );
+			return [ { key: '', label: this.props.translate( 'Loading…' ), disabled: 'disabled' } ];
 		}
+		return countriesList.map( ( { code, name }, idx ) => (
+			{
+				key: idx,
+				label: name,
+				code,
+			}
+		) );
+	},
+
+	render() {
+		const countriesList = this.props.countriesList.get(),
+			options = this.getOptions( countriesList );
 
 		return (
 			<select
```

Comment:
We could have, and probably should have. I just touch the component enough to fix the bugs and eslint